tgui/packages/tgui/interfaces/common/WirePanel/Wire.tsx

```diff
@@ -0,0 +1,73 @@
+/**
+ * @file
+ * @copyright 2023
+ * @author glowbold (https://github.com/pgmzeta)
+ * @license MIT
+ */
+
+import { RemoteAccessButton } from ".";
+import { LabeledList } from "../../../components";
+import { WirePanelAction } from "./const";
+import { WireData } from "./type";
```

Comment:
```suggestion
import type { WireData } from "./type";
```
Not a requirement, just a nice annotation that "guarantees" this is only type data so will be erased when building.